tests/testthat/test-methods.R

```diff
@@ -128,15 +122,15 @@ test_that("print.lint works for inline data, even in RStudio", {
 
 test_that("print.lints works", {
   withr::local_options(lintr.rstudio_source_markers = FALSE)
-  tmp <- withr::local_tempfile()
 
-  expect_true(file.create(tmp))
-  expect_invisible(print(lint(tmp)))
+  withr::with_tempfile("tmp", code = {
```

Comment:
The `local_tempfile()` approach seems preferrable to me because it explicitly sets `tmp`.

Reply:
I have reverted this change.